packages/theme-ui/src/provider.js

```diff
@@ -0,0 +1,67 @@
+import { ThemeContext as EmotionContext } from '@emotion/core'
```

Comment:
This file was renamed, previously `core.js` – must've missed the `git mv` or something